### PR TITLE
Use the new switch expression

### PR DESCRIPTION
**Pull request by Feichtmeier**

### CODE REVIEW

File: lib/app/settings/settings_page.dart [hunk 1]
```diff
@@ -154,28 +142,15 @@ class ThemeSection extends StatefulWidget {
...
-          {
-            App.themeNotifier.value = ThemeMode.dark;
-          }
-          break;
+  void onChanged(int index) {
+    for (var themeMode in ThemeMode.values) {
+      if (themeMode.index == index) {
+        setState(() {
+          App.themeNotifier.value = themeMode;
+        });
```
Comment (jpnurmi) [inline lib/app/settings/settings_page.dart:150]:
```dart
setState(() {
  App.themeNotifier.value = ThemeMode.values[index];
});
```

File: lib/app/common/app_format.dart [hunk 2]
```diff
@@ -24,14 +24,10 @@ enum AppFormat {
...
   String localize(AppLocalizations l10n) {
-    switch (this) {
-      case AppFormat.snap:
-        return l10n.snapPackages;
-      case AppFormat.packageKit:
-        return l10n.debianPackages;
-      default:
-        return '';
-    }
+    return switch (this) {
```
Comment (jpnurmi) [inline lib/app/common/app_format.dart:27]: looks subjectively better than the floating indented switch above and below

**Review by jpnurmi — APPROVED** [review]
LGTM :+1:

File: lib/app/settings/settings_page.dart (hunk 1 above)
Comment (Feichtmeier) [reply lib/app/settings/settings_page.dart:150]: Oh 🤯 you're right. Now it's merged. I'll do a continuous pr

File: lib/app/common/app_format.dart (hunk 2 above)
Comment (Feichtmeier) [reply lib/app/common/app_format.dart:27]: ah you mean the return thing is better then the expression body because the lines are not that long this way?

**Pull request merged**